### PR TITLE
add autspaces feedback summary

### PR DESCRIPTION
**Pull request by soledadli**

### CODE REVIEW

**Comment by BrainonSilicon** [comment]
Hey @soledadli! Thank you so much for capturing all this information!! 

Would you be able to update the README to reflect the added file? I'm trying to wrange the READMEs a little more so that they explain what is in that specific folder. 

The README should be changed to something like this: 

# Design Research
This folder acts to collate and point to information and feedback about the design of the AutSPACEs platform and associated projects. If you are searching for the platform itself, please visit the accompanying repository: [AutSPACEs](https://github.com/alan-turing-institute/AutSPACEs). 

## Contents
* [Personas](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/platform-designs/design-research/Personas)
* [feedback-linkspad.md](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/platform-designs/design-research/feedback-linkspad.md)
* `<YOUR NEW FILE>`
* you can find a good example of this layout [here](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/accessibility/README.md)

## How to Contribute
There are many ways to get involved with this project and we would love to hear your thoughts!

If you would like to contribute to this repository, please read the [Contributors Guidelines](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/.github/CONTRIBUTING.md) and familiarize yourself with the project and this repository. You can also join our meetups which happen on the first and last thursday of every month (find out when the next one is happening by signing up to the mailing list [here](https://tinyletter.com/AutisticaTuringCitizenScience), or reach out to the team.

**Review by BrainonSilicon — CHANGES_REQUESTED** [review]
Hi @soledadli -  I left a comment about the README file that should be updated alongside the file. 

It's probably good housekeeping to create the habit of updating the READMEs as files are added (instead of post-hoc). If you have time, can you update the README and then we can merge the PR.

**Comment by soledadli** [comment]
@BrainonSilicon Hi Sophia. I updated the README. Let me know whether the information is clear now or not.

File: platform-designs/design-research/README.md [hunk 1]
```diff
@@ -1,4 +1,15 @@
 # Design Research 
 
-This folder acts to collate and point to information and feedback about the Design of the platform.
+This folder acts to collate and point to information and feedback about the Design of the platform. If you are searching for the platform itself or the platform development code, please visit the accompanying repository: [AutSPACEs](https://github.com/alan-turing-institute/AutSPACEs).
 
+## Contents
+* [Personas](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/platform-designs/design-research/Personas)
+* [feedback-linkspad.md](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/platform-designs/design-research/feedback-linkspad.md)
+* [feedback-summary(2021-10-09)](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/feedback-summary-1/platform-designs/design-research/AutSPACES%20Feedback%20(19.10).pdf)
+  * The summary identified the most needed needs for design, functions, and contents from feeback for the previous prototypes. Detailed information can be found the feedback summary itself.
```
Comment (BrainonSilicon) [inline platform-designs/design-research/README.md:9]: possible change 
> `identified the most needed needs for design` 

to 

> `identified the "key" user needs for the platform's design, function, and contents based on feeback for the previous prototypes.`

File: platform-designs/design-research/README.md [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+This folder acts to collate and point to information and feedback about the Design of the platform. If you are searching for the platform itself or the platform development code, please visit the accompanying repository: [AutSPACEs](https://github.com/alan-turing-institute/AutSPACEs).
 
+## Contents
+* [Personas](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/platform-designs/design-research/Personas)
+* [feedback-linkspad.md](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/platform-designs/design-research/feedback-linkspad.md)
+* [feedback-summary(2021-10-09)](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/feedback-summary-1/platform-designs/design-research/AutSPACES%20Feedback%20(19.10).pdf)
+  * The summary identified the most needed needs for design, functions, and contents from feeback for the previous prototypes. Detailed information can be found the feedback summary itself. 
+* you can find a good example accessibility research [here](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/accessibility/README.md)
```
Comment (BrainonSilicon) [inline platform-designs/design-research/README.md:10]: Suggested capitalization change of : 
> `you ...`

to 

> `You`

**Review by BrainonSilicon — COMMENTED** [review]
I can leave it up to you if you would like to address the changes ⭐

**Review by BrainonSilicon — APPROVED** [review]
Awesome! 

LGTM 🚀 🚀

**Pull request merged**